src/bindings/js/node/include/model_wrap.hpp

```diff
@@ -108,6 +108,19 @@ class ModelWrap : public Napi::ObjectWrap<ModelWrap> {
      * @return Napi::Array containing a shape of requested output.
      */
     Napi::Value get_output_shape(const Napi::CallbackInfo& info);
+    
+    /**
+     * @brief Helper function to access model output elements types.
+     * @return Napi::String representing the element type of the requested output.
+     *
+     */
+    Napi::Value get_output_element_type(const Napi::CallbackInfo& info);
+
+    /**
+     * @brief Helper function to access model output elements types.
+     * @return Napi::String representing the element type of the requested output.
+     *
+     */
```

Comment:
The method description should be placed only once, above the declaration. This one can be deleted.
```suggestion
```